cookbook/controller/error_pages.rst

```diff
@@ -5,172 +5,154 @@
 How to Customize Error Pages
 ============================
 
-When an exception is thrown, the core ``HttpKernel`` class catches it and
-dispatches a ``kernel.exception`` event. This gives you the power to convert
-the exception into a ``Response`` in a few different ways.
+In Symfony applications, all errors are treated as exceptions, no matter if they
+are just a 404 Not Found error or a fatal error triggered by throwing some
+exception in your code.
 
-The core TwigBundle sets up a listener for this event which will run
-a configurable (but otherwise arbitrary) controller to generate the
-response. The default controller used has a sensible way of
-picking one out of the available set of error templates.
+In the `development environment`_ Symfony catches all the exceptions and displays
+a special **exception page** with lots of debug information to help you quickly
+discover the root problem.
 
-Thus, error pages can be customized in different ways, depending on how
-much control you need:
+.. image:: /images/cookbook/controller/error_pages/exceptions-in-dev-environment.png
+   :alt: A typical exception page in development environment
 
-#. :ref:`Use the default ExceptionController and create a few
-   templates that allow you to customize how your different error
-   pages look (easy); <use-default-exception-controller>`
+Since these pages contain a lot of sensitive internal information about your
+application, in production environment Symfony displays instead a simple and
```

Comment:
This is a very big sentence, let's break it up: "Since these pages contain a lot of sensitive internal information, Symfony won't display this in the production environment. Instead, it'll show a simple and generic **error page**."